app/services/foreman_gutterball/gutterball_service.rb

```diff
@@ -26,26 +26,12 @@ class GutterballService < GutterballResource
     def initialize(_options = {})
     end
 
-    def available_reports
+    def reports(report_key = nil, query_params = nil)
```

Comment:
i would suggest using `request.query_parameters` instead and doing some manipulation on the specific params that we want to pass on to our call to the gutterball API.